README.md

```diff
@@ -1,3 +1,14 @@
+1. A concern I have is that the various indexer implementations are not exactly equivalent in terms of sophistication 
+    - For example, the `subgraph`, `ponder`, and `envio` implementations simply write events to their respective data stores, which is then polled by the frontend - totally fine and practical for our purposes
+    - However, the `qn-streams`, `thirdweb-insights-api`, `goldsky-mirror` and `allium-ws-server` implementations take things a step further by exposing websocket servers which can broadcast messages to clients, which means the frontend doesn't need to poll the data store
+    - This is of course a really great approach and should be our recommendation for building high performance apps, but I don't sense it's valid to compare these to the simpler implementations mentioned above
```

Comment:
Yeah I feel like it makes sense to separate these in the UI / folders. Something like

realtime/ vs poll/